src/jsSSE.php

```diff
@@ -41,12 +39,10 @@ public function init(): void
      * A function that get execute when user aborted this sse.
      *
      * @param callable  $fx
-     * @param mixed     $param
      */
-    public function onAborted(callable $fx, $param)
+    public function onAborted(callable $fx)
```

Comment:
Should be removed, user can use `$this->onHook('aborted', $fx);` directly wih the full flexibility like priority.

Reply:
It does not prevent from using  `$this->onHook('aborted', $fx);` This is only a shortcode method. 
I believe it is easier for a new user for Atk when you get a direct function like this.

Reply:
It only duplicates code and any change is there much more painfull to introduce. I have checked the other hooks and method like this is not common, so it should be removed for consistency anyway :)

Reply:
Do not agree with you on this one. This is a design choice.

I really think it makes the developer, especially the new ones, to be aware of the method. 
It is easy to know that method exists by looking at the code or when using code completion

The other advantage is that it makes it more backward-compatible.
 Form::onSubmit() is a good case. Imagine all the break we would have introduced when we changed the method name from addHook to onHook if people where using addHook('submit') instead of onSubmit.

Reply:
Please do not duplicate any code, this is really not needed.

Reply:
i agree with ibelar, some class needs helper functions, IMHO is not a duplication.

Reply:
@ibelar I still think this is wrong, but as the hook names needs to be put in constants everywhere, I have nothing againt this PR and actually 👍 for it!

I would also avoid comments like "Sending an sse action." when it is clear what the method does. It actually never helps, as the comments are almost never up too date and if programmer need to know more than "send()", then he goes to the method impl. code anyway.